crates/sui-indexer/migrations_v2/2023-08-19-044023_objects/up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE objects (
+    object_id                   bytea         PRIMARY KEY,
+    object_version              bigint        NOT NULL,
+    object_digest               bytea         NOT NULL,
+    checkpoint_sequence_number  bigint        NOT NULL,
```

Comment:
Objects don't seem to have an edge connecting them to the transaction that created them (i.e. `previousTransaction`).

Reply:
`previousTransaction` is part of `Object` which is bcs serialized in another field. When we query a tx we get its updated objects from `transaction` table. Are you asking this because client may query `previous_transaction` and only it?

Reply:
Got it -- yes, that's one of the fields that can be independently queried.  It's also one of the `ObjectFilter`-s.

Generally, having to deserialize the BCS to get access to some fields is not the end of the world.  For the second point, how easy would it be to run this query by querying the transaction table for the previous transaction, and then get the IDs of its changed objects?  If that seems like a reasonable strategy, then we can avoid the extra column.

Reply:
That would just require another round-trip through the db, not the end of the world and we can optimize things as we go as well

Reply:
> how easy would it be to run this query by querying the transaction table for the previous transaction, and then get the IDs of its changed objects?

transaction table is designed to support this query, there's a `object_changes` field there. something like
```
select object_changes from transactions where transaction_digest = '';
```